docs/motivation.md

```diff
@@ -235,44 +234,56 @@ via this new API – since clearly there are some cases where strong validation
 is needed (e.g., dynamically being able to remove a linecard would mean also
 invalidating many other entities that depend on that linecard and its
 interfaces - so it does not seem a good candidate for the mechanisms
-corresponding to dynamic updates described in this document). Therefore, we
-propose that where payloads are generated for the `Modify` RPC, we give these a
-new place within the OpenConfig schema, particularly, defining a new
-`volatile`/`dynamic`/`ephemeral` subtree to correspond with the `config` and
+described in this document). Therefore, we propose that where payloads are
+generated for the `Modify` RPC, we give these a new place within the OpenConfig
+schema, particularly, defining a new
+`volatile`/`ephemeral` subtree to correspond with the `config` and
```

Comment:
Maybe the design decisions for modeling should be outside the scope of the motivation doc?

That said, would an annotation be more appropriate?

Reply:
They can be moved, yes.

I'm not sure how an annotation would work here -- what are you proposing? There are other PRs that cover how the content of these RPCs are proposed to be generated. PTAL at #6.

Reply:
The volatile container could have an annotation so there is a property in yang identifying the node as volatile. 
It might be useful so yang aware code can identify volatile leafs programmatically versus relying on a "magic" container name.   (Although I think using "volatile" as a container name also makes sense from a style perspective.  I would do both the annotation an the container name)  

An analogous entity is using the [yang config statement](https://datatracker.ietf.org/doc/html/rfc6020#section-7.19.1), but also as an OC style guide, we use a container named 'config'.

Reply:
Ah, I see -- yes, annotating as well as a container name makes sense. I think we must have a container name for consistency. The alternate suggestion of having an annotation on an existing `config` leaf would be possible but add more complexity of magic in the tooling to know that such an annotation means that there is another path for the leaf.

Reply:
Ok, agreed that we should have both:  a container and an annotation such as `config-volatile`, as this is the most explicit.